rust/gui-client/src-common/src/logging.rs

```diff
@@ -49,11 +48,16 @@ pub enum Error {
 /// `Clone` yet, so that's why we take the directives string
 /// <https://github.com/tokio-rs/tracing/issues/2360>
 pub fn setup(directives: &str) -> Result<Handles> {
+    if let Err(error) = output_vt100::try_init() {
+        tracing::debug!("Failed to init terminal colors: {error}");
+    }
+
     let log_path = known_dirs::logs().context("Can't compute app log dir")?;
 
-    // Logfilter for stderr cannot be reloaded. This is okay because we are using it only for local dev and debugging anyway.
+    // Logfilter for stdout cannot be reloaded. This is okay because we are using it only for local dev and debugging anyway.
```

Comment:
Digging into `tracing`, i realised our default is actually stdout! I'll make a follow-up to change to stderr.